### DISCUSSION TITLE
Global/collection notification banner in the header.

### DISCUSSION THREAD

**Discussion by vivekpulukuri** [specify/specify7#5823] [Feature Requests]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
A sticky messgae, a loud banner on the page  in the header to grab user's attention.
Confugurable by the admin at collection and global levels 
example uses cases
- Upcoming outages/upgrades
- Any immediate message/warning/caution of and etc to all users
- To mark logged in env DEV/TEST/TRAINING and etc
- Important reminders 
- links to help pages and etc

**Reported By**
CSIRO

**Additional context**
Something like this?
![spbanner](https://github.com/specify/specify7/assets/2975626/ed43c1cb-5e12-4e99-9f91-f6a06fa4e14e)

**Comment by maxpatiiuk** [comment]
Nice idea!

At one point we had an option to embed an arbitrary webpage as an iframe instead of the welcome page image, but that was disabled as it was confusing to the average user (i.e, due to security rules in modern browsers, the webpage you are trying to embed must explicitly allow being embedded before it can be displayed like that - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options - as a result most of the pages a regular user tried didn't work)

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/maintenance-mode-in-specify-7/2092/2

**Comment by philippeverley** [comment]
Top banner would definitely be useful to both CAY and NOU instances.
+1
Thanks
